### ISSUE TITLE
Unmark files

### ISSUE THREAD

**Issue by kchousos** [karthink/dired-delight#1]
Hello karthink, thanks for this handy package. I find it very useful, but I cannot find a way to unmark a file after I've marked it. Is there a way to do that? Many thanks.

**Comment by karthink** [comment]
Hi @kchousos, you just call `dired-delight` again on the file(s) but don't supply a color, i.e. leave it blank. You may have to refresh the display (`g` in dired) afterwards to see the color removed.

May I ask how you use this feature? I wrote it since someone requested it on reddit and it was quite simple to implement, but I don't quite know how to use it myself.

**Comment by kchousos** [comment]
Right, I tried that but one of the two files wasn't unmarked even after refreshing the buffer. Maybe there was a mistake in my part.

As far as use cases go, I also recently searched for such a functionality in fired since I wanted to save my position in a series of lecture pdfs. My envisioned workflow would be to permanently mark the pdf I last read, so I can jump into it the next time I study. I also wanted this to be a directory-context thing, instead of saving normal emacs bookmarks for every pdf in every directory I'm currently on.

**Comment by karthink** [comment]
> Right, I tried that but one of the two files wasn't unmarked even after refreshing the buffer. Maybe there was a mistake in my part.

I see. I've pushed an update since you opened the issue. If you still have this problem after updating (with a fresh pair of files) and can tell me how to reproduce it I should be able to fix it.

> I wanted to save my position in a series of lecture pdfs. My envisioned workflow would be to permanently mark the pdf I last read, so I can jump into it the next time I study. 

Another interesting use case I wouldn't have thought of. Behind the scenes this is also a (very) simplified bookmark system, only the front end is colored blocks instead of a list or menu.

**Comment by dasoju** [comment]
I ran into same issue.
In my case, when i use @. it pops up helm interface with all the colors as below 
![image](https://github.com/karthink/dired-delight/assets/54728201/62dcfcf0-6472-4191-af34-10cb5d13cc5f)

I entered "" as below
![image](https://github.com/karthink/dired-delight/assets/54728201/cb623e0c-0940-4c56-9241-1169290463bd)

But i did not see it removing the color tab 
I also tried just blank but still didn't work

Perhaps, since there is no match for "" or blank in the helm, there is no effect.
I think this needs tobe fixed

**Comment by karthink** [comment]
@dasoju I don't know how Helm handles blank input -- with ido/icomplete there's a special key you have to press to confirm your input verbatim. Could you try without Helm and let me know?
